pkg/nodebootstrap/assets/bootstrap.al2.sh

```diff
@@ -1,7 +1,31 @@
-#!/bin/sh -eu
+#!/bin/bash -e
 
-echo "NODE_IP=$(curl --silent http://169.254.169.254/latest/meta-data/local-ipv4)" > /etc/eksctl/kubelet.local.env
-echo "INSTANCE_ID=$(curl --silent http://169.254.169.254/latest/meta-data/instance-id)" >> /etc/eksctl/kubelet.local.env
+function get_max_pods() {
+  MAX_PODS_FILE="/etc/eksctl/max_pods_map.txt"
+  while read instance_type pods; do
+
+    if  [[ "${instance_type}" == "${1}" ]] && [[ "${pods}" =~ ^[0-9]+$ ]] ; then
+      echo ${pods};
+      return
+    fi ;
+
+    done < "${MAX_PODS_FILE}"
```

Comment:
Should we drop indentation here... Also, not quite sure about having `<` at the end here, rather then at the top.

Reply:
Will fix the indentation. the `<` has to go there AFAIK. It doesn't work in the top.

Reply:
Yes, looks like this is the canonical way to do it :)